File: submissions/CodesRN/problem3.java
import java.util.*;
class problem3
{
    public static void main(String args[])
    {
        int sum=0;
        int sum1=0;
        Scanner sc=new Scanner(System.in);
        //System.out.println("Enter a number");
        int m=sc.nextInt();
        if(m>2 && m<10)
        {   int m1[][]=new int[m][m];
            int k1;
            int suml=0;int sumr=0;
          //  System.out.println("Enter "+(m*m)+" numbers");
            for(int i=0;i<m;i++)
            {
                for(int j=0;j<m;j++)
                m1[i][j]=sc.nextInt();
            }
              for(int i=0;i<m;i++)
            { suml=suml+m1[i][i];
            }
           // System.out.println(suml);
                k1=0;
               int k2=m-1;
            while(k1!=m)
            {
                sumr=sumr+m1[k1][k2];
                k1=k1+1;
                k2=k2-1;
            }
           // System.out.println(sumr);
            if(suml>sumr)
           System.out.println(suml);
          else
            System.out.println(sumr);
        }
        else
        System.out.println("SIZE OUT OF RANGE");
    }
}

File: submissions/superman/problem3.cpp
#include<stdio.h>
main()
{
	int a[10][10],m,suml=0,sumr=0,f=0,i,j;
	scanf("%d",&m);
	if(m<2 || m>10)
	{
		printf("SIZE OUT OF RANGE");
		f=1;
	}
	if(f==0){
	
	for(i=0;i<m;i++)
	{
		for(j=0;j<m;j++)
		{
			scanf("%d",&a[i][j]);
		}
	}
	for(i=0;i<m;i++)
	{
		for(j=0;j<m;j++)
		{
			if(i==j)
			{
				suml=suml+a[i][j];
			}
		}
	}
	for(i=0;i<m;i++)
	{
		for(j=0;j<m;j++)
		{
		
			if((i+j)==m-1)
			{
			
				sumr=sumr+a[i][j];
		}
		}
	}
	if(sumr>suml)
	{
		printf("%d",sumr);
	}
	else
	printf("%d",suml);
	
}}